### DISCUSSION TITLE
Thanks for your suggestions

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2110] [General]
@Lupum1001
Looks like you are quite competent.  It's great to have you.
I was impressed that you have identified a dormant file (440 lines) we can delete.

We have 152 open issues.  Some of them are quite annoying and serious items.  Would you be so kind to have a look and see if you can help us with any of them?  
Often, newcomers come and have a proposal for a new feature, or a modification to make things more clear in their opinion.  Their PR gets merged and we never hear from them again.   But, who is going to fix the existing problems then?  Specially the ones related to their feature?

Can you think of a way we can encourage developers, old and new, to help fix the existing issues instead of focusing on what else can be done with xDrip? 

Everything I have written here is my personal opinion.  It is not from the developers.

**Comment by tolot27** [comment]
@Lupum1001 Maybe you can have a look at #1781. This is related to the settings. And #1522 is a big task list to cleanup the code.

**Comment by Lupum1001** [comment]
Thanks, that's very kind of you @Navid200.

My initial focus atm is working on xDrip's UI/UX, and I may be able to dig deeper into the weeds on a few things like Dexcom functionality. I'm brand new to Android development though, so I expect it'll take me awhile before I'm comfortable helping with things like https://github.com/NightscoutFoundation/xDrip/issues/1522.

Speaking for myself, I'm generally motivated to work on xDrip issues that I encounter on a day-to-day basis, and I assume that most other newcomers will be of a similar mindset. 
I think @tolot27 has the right idea, which is to just let newcomers work on what interests them, and point them towards related bugs and see if they bite. I'll bookmark https://github.com/NightscoutFoundation/xDrip/issues/1781 in case there's anything I can do.

If unaddressed bugs are a big problem, you might be able to set up a bounty program, where everyday users can stake a few dollars on  bugs that they feel aren't getting fixed fast enough. I noticed a few sites like BountySource.com and GitPay.me, but I've no idea what the best options are.
